testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -782,6 +782,17 @@ protected static SortedSet<String> keywords(@Nullable String dialect) {
             + "FROM UNNEST((ARRAY[1, NULL, 3, 4])) AS x");
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-5996">[CALCITE-5996]</a>
+   * TRANSLATE operator is incorrectly unparsed. */
+  @Test void testTranslate() {
+    sql("select translate(col using utf8)\n"
```

Comment:
I think it would be better if you move this test case to `testConvertAndTranslate()`.